common/lib/xmodule/xmodule/js/karma_xmodule.conf.js

```diff
@@ -77,7 +78,8 @@ var options = {
     ],
 
     fixtureFiles: [
-        {pattern: 'fixtures/*.*'}
+        {pattern: 'fixtures/*.*'},
```

Comment:
`fixtures/*.*` is not loading hls video files present under `fixtures/hls/` directory. So added a new file pattern.